notebooks/thunder_as_torch_compile_backend_tutorial.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    # Create the ThunderCompiler backend
</pre><p> </p><p>Nit: comments are descriptive, so "Creates the ThunderCompiler backend" is easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>